### DISCUSSION TITLE
Shortcut to any menu page with MIDI notes

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#675] [Ideas]
Menu diving could be faster if MIDI notes was used to access any page. 

For exemple, if you tweak the reverb everytime during a performance, you could add a MIDI note to access the reverb page with only one push of a button.

At least 8x shortcuts should be added

**Comment by probonopd** [comment]
I could imagine that we define one pin on the PRi as a "function" button.
When pressed, the keys on the attached MIDI keyboard would all have dedicated functions. (You would have to remember or use labels to know which MIDI keyboard key would mean which function.)
When you press that key again, the MIDI keyboard world go back to its usual function.

Could be done, but is possibly complicated to implement given our current menu driven operation. Wdyt @diyelectromusic?

**Reply by diyelectromusic** [reply]
We allow MIDI notes to be used for navigation but not for shortcuts... I mean it's all possible, but the configuration might start to get a bit complicated! :)

Kevin

**Reply by Onosendaifr** [reply]
What about this : 

Right now, to access a page you need to press Select button with a quick press. What about adding a long press to activate MIDI learn (info on the screen), then you just have to press the button you want on your controller to create the shortcut to this specific page. I've tried, a long press don't activate the page.

Then you can add a new Shortcut page (after Performance), this page will list all shortcut created and you can delete them if needed
